packages/devtools_app/lib/src/screens/profiler/cpu_profile_controller.dart

```diff
@@ -551,11 +531,13 @@ class CpuProfilerController
     resetSearch();
   }
 
+  @override
   void dispose() {
     _dataNotifier.dispose();
     _selectedCpuStackFrameNotifier.dispose();
     _processingNotifier.dispose();
     transformer.dispose();
+    super.dispose();
```

Comment:
Nit: `super.dispose()` should probably be the first call in this method.

Reply:
dispose is the one exception where we want to dispose last so that a subclass's disposable items are disposed before the superclass. Most of our codebase follows this pattern, but we probably need to clean up a few cases for consistency.